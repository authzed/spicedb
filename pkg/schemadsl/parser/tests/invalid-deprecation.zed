definition deprecated_relation {
    @deprecated()
    relation reader: user
}

definition user {}