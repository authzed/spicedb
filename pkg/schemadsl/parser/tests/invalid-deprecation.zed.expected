NodeTypeFile
  end-rune = 48
  input-source = invalid deprecated relation test
  start-rune = 0
  child-node =>
    NodeTypeDefinition
      definition-name = deprecated_relation
      end-rune = 48
      input-source = invalid deprecated relation test
      start-rune = 0
      child-node =>
        NodeTypeDeprecation
          end-rune = 48
          input-source = invalid deprecated relation test
          start-rune = 37
          child-node =>
            NodeTypeError
              end-rune = 48
              error-message = Expected identifier, found token TokenTypeRightParen
              error-source = )
              input-source = invalid deprecated relation test
              start-rune = 49
        NodeTypeError
          end-rune = 48
          error-message = Expected end of statement or definition, found: TokenTypeRightParen
          error-source = )
          input-source = invalid deprecated relation test
          start-rune = 49
    NodeTypeError
      end-rune = 48
      error-message = Unexpected token at root level: TokenTypeRightParen
      error-source = )
      input-source = invalid deprecated relation test
      start-rune = 49
